src/schema/me/__tests__/order/update_order_mutation.test.js

```diff
@@ -0,0 +1,159 @@
+import { runAuthenticatedQuery, runQuery } from "test/utils"
+import { readFileSync } from "fs"
+import { resolve } from "path"
+import exampleOrderResult from "test/fixtures/results/order_mutation"
+
+describe("Me", () => {
+  describe("UpdateOrderMutation", () => {
+    const ordersPath = resolve(
+      "src",
+      "test",
+      "fixtures",
+      "gravity",
+      "order.json"
+    )
+    const order = JSON.parse(readFileSync(ordersPath, "utf8"))
```

Comment:
I _think_ `import someJSON from '/path/to/json'` works out of the box?

Reply:
That does work! Good catch. I wonder why we don't use this pattern elsewhere in metaphysics.